build/chip/chip_test_suite.gni

```diff
@@ -93,13 +220,17 @@ template("chip_test_suite") {
     }
 
     deps = [ dir_pw_unit_test ]
+    if (enable_efr32_build_per_suite) {
```

Comment:
This `if` branch is not executed unless we change the debug flags to enable per-suite binaries instead of per-test binaries.